meson.build

```diff
@@ -1,4 +1,4 @@
-project('snapd-desktop-integration', 'c', version: '0.9')
+project('snapd-desktop-integration', 'c', version: '0.9raa1')
```

Comment:
Was this just for debugging?

Reply:
Ops! No, it was for the testing channel in the store. But maybe we should bump the version number to 0.10...